code/modules/mob/living/simple_animal/bot/floorbot.dm

```diff
@@ -41,12 +42,16 @@
 	#define REPLACE_TILE		6
 	#define TILE_EMAG		7
 
-/mob/living/simple_animal/bot/floorbot/Initialize()
+/mob/living/simple_animal/bot/floorbot/Initialize(mapload, toolbox_color)
```

Comment:
rename the toolbox_color param (perhaps `new_toolbox_color` or something), then remove the src. stuff below please.